gcp_variant_transforms/testing/integration/preprocessor_tests/no_conflicts.json

```diff
@@ -2,7 +2,7 @@
   "test_name": "no-conflicts",
   "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/valid-4.0.vcf",
   "blob_name": "temp/report_no_conflicts.tsv",
-  "runner": "DataflowRunner",
+  "runner": "DirectRunner",
```

Comment:
I'm assuming these are leaking from the other PR? In any case, please remove them from this PR as they're unrelated to this change.